wire-ios-utilities/Source/Sequence+Async.swift

```diff
@@ -64,4 +54,11 @@ extension Sequence {
         return values
     }
 
+    /// Please try to use a loop instead.
```

Comment:
?

Reply:
In my opinion there is no need to extend Apple's collection API.
Instead of adding all those little helpers where we are not sure about side effects, why not use a regular for-loop instead?

❌
```
await deletedClients.asyncForEach {
    await $0.deleteClientAndEndSession()
}
```

✅
```
for deletedClient in deletedClients {
    await deletedClient.deleteClientAndEndSession()
}
```

Reply:
I'll add the comment to PR https://github.com/wireapp/wire-ios/pull/791

Reply:
removed: 4e64690491cb3b8e7b11f8408765a780c8a258d0